packages/manager-react-components/package.json

```diff
@@ -35,6 +35,7 @@
     ]
   },
   "dependencies": {
+    "@ovh-ux/manager-common-translations": "*",
```

Comment:
nitpick(blocking): We cannot `@ovh-ux/manager-common-translations` as a direct dependency. Can you add this package in `devDependencies` and `peerDependencies`?